lib/modules/dosomething/dosomething_organ_donation/dosomething_organ_donation.module

```diff
@@ -22,9 +22,17 @@ function dosomething_organ_donation_menu() {
     'type' => MENU_CALLBACK,
   ];
 
+  $items['organ-donation/post-registration'] = [
+    'page callback' => 'dosomething_organ_donation_post_registration',
+    'access callback' => 'user_access',
+    'access arguments' => array('administer actions'),
+    'type' => MENU_CALLBACK,
+  ];
+
   $items['organ-donation/send-postal-code'] = [
```

Comment:
two other minor nitpick things. 
let's call these menu items
`organ-donation/postal-code` and `organ-donation/registration`